templates/model/templates/affiliated_committer_detail.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
the base layout of these html pages should be one template that other the other templates re-use